propagation/SWALS/src/util/netcdf_util_mod.f90

```diff
@@ -36,8 +36,8 @@ module netcdf_util
     character(len=charlen), parameter :: gridded_output_variables_time(4) = &
         [character(len=charlen) :: 'stage', 'uh', 'vh', 'elev']
         !! Gridded variables that can be output as time-series. Must match [STG, UH, VH, ELV] in order.
-    character(len=charlen), parameter :: gridded_output_variables_max_U(5) = &
-        [character(len=charlen) :: 'max_stage', 'max_speed', 'max_flux', 'arrival_time', 'time_of_max_stage']
+    character(len=charlen), parameter :: gridded_output_variables_max_U(6) = &
+        [character(len=charlen) :: 'max_stage', 'min_stage', 'max_speed', 'max_flux', 'arrival_time', 'time_of_max_stage']
```

Comment:
Could you please move `min_stage` to the end of this array? It probably doesn't matter. But the comment mentions assumptions that `max_stage` is first. I would not be shocked if somewhere there exists a model which assumes that `max_speed` is second. If we add new variables to the end, then problems seem less likely.

Reply:
Resolved in [3c42c55](https://github.com/GeoscienceAustralia/ptha/pull/49/commits/3c42c55ca7e02277ca38b3c511d4a5304fc634e7)